persistence.js

```diff
@@ -21,6 +21,16 @@ var OUTGOING = 'outgoing:'
 var OUTGOINGID = 'outgoing-id:'
 var INCOMING = 'incoming:'
 var WILL = 'will:'
+var encodingOption = {
```

Comment:
Could you transform all this including modules require to `const`?

Reply:
Sure, no problem. I was thinking about this as well.